### DISCUSSION TITLE
Opening performance: live-coded TidalCycles + Hydra

### DISCUSSION THREAD

**Discussion by tsadarsh** [github/india#70] [Satellite India 2021 Q&A]
I'm loving this performance :heart:. Can this be said as procedural music? I'm also curious to learn these skills. It is just mind blowing to see such amazing performances.

**Comment by tsadarsh** [comment]
Hey this sounds like Punjabi beats.Yay! :smile: 
Please keep this beat going!

**Reply by khoparzi** [reply]
Hey @tsadarh, thanks for all the love.

You could say parts of this were procedural, specially the dayan on the dhol. The other parts were done with deterministic methods. But it always helps to mix it up 😊

**Comment by AlaineN** [comment]
This duo performed at GitHub Universe as well! They're awesome. You can find more of their stuff here:
https://khoparzi.com/aboutme/     
https://www.instagram.com/khoparzi/

https://samarthgulati.com/     
https://twitter.com/samarthishere

**Reply by khoparzi** [reply]
Thanks for the links @alaine 🙌🏽

I'll put up the code for my performance later today

**Comment by khoparzi** [comment]
Here is all my code from today https://gist.github.com/khoparzi/1d886914874d5fb68173c4202e2daced

**Reply by tsadarsh** [reply]
I should begin to learn this wizardry. This is simply too cool. Thanks @khoparzi :heart:

**Reply by khoparzi** [reply]
You’re welcome man. Do throw questions my way if you get stuck.
